gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest_asyncio.py.j2

```diff
@@ -166,14 +169,18 @@ class Async{{service.name}}RestTransport(_Base{{ service.name }}RestTransport):
 
             {% if not method.void %}
             # Return the response
+            {% if method.server_streaming %}
+            resp = rest_streaming_async.AsyncResponseIterator(response, {{method.output.ident}})
+            {% else %}
             resp = {{method.output.ident}}()
             {% if method.output.ident.is_proto_plus_type %}
             pb_resp = {{method.output.ident}}.pb(resp)
             {% else %}
             pb_resp = resp
-            {% endif %}
+            {% endif %}{# if method.output.ident.is_proto_plus_type #}
```

Comment:
One question: we handle proto_plus explicitly in the non-streaming case here. But what about in streaming results? Looking at api-core's `_rest_streaming_base.py:BaseResponseIterator.grab()`, it's not obvious that we do. Do we? Should we? Is this question irrelevant?

Reply:
I've filed https://github.com/googleapis/gapic-generator-python/pull/1428 to Investigate this later.